ProjectLighthouse.Servers.GameServer/Controllers/Login/LoginController.cs

```diff
@@ -130,6 +130,23 @@ public async Task<IActionResult> Login()
                 Logger.Warn($"Unknown user tried to connect username={username}", LogArea.Login);
                 return this.Forbid();
             }
+
+            // Block RPCN signups if forbidden in config
+            if (npTicket.Platform == Platform.RPCS3 && !ServerConfiguration.Instance.Authentication.AllowRPCNSignup)
+            {
+                Logger.Warn(
+                    $"New user tried to sign up via RPCN, and that is forbidden in the config, username={username}, remoteIpAddress={remoteIpAddress}",
+                    LogArea.Login);
+            }
+
+            // Block PSN signups if forbidden in config
+            if (npTicket.Platform == Platform.RPCS3 && !ServerConfiguration.Instance.Authentication.AllowPSNSignup)
```

Comment:
Shouldn't this be checking for `Platform.PS3` `Platform.Vita`, and `Platform.PSP`

<sub><sup>oops</sup></sub>

Reply:
Yeah, I fixed that previously, it now refers to an extension function of the platform enum that checks if it's PSN or not